tests/libnode/node.go

```diff
@@ -351,3 +360,31 @@ func SetNodeUnschedulable(nodeName string, virtCli kubecli.KubevirtClient) {
 func SetNodeSchedulable(nodeName string, virtCli kubecli.KubevirtClient) {
 	setNodeSchedualability(nodeName, virtCli, true)
 }
+
+func GetVirtHandlerClient(virtCli kubecli.KubevirtClient) kubecli.VirtHandlerClient {
+	namespace, err := kubevirtutil.GetNamespace()
+	util.PanicOnError(err)
+
+	clientConfig, err := kubecli.GetKubevirtClientConfig()
+	util.PanicOnError(err)
+
+	aggregatorClient := aggregatorclient.NewForConfigOrDie(clientConfig)
+
+	kubeInformerFactory := controller.NewKubeInformerFactory(virtCli.RestClient(), virtCli, aggregatorClient, namespace)
+	authConfigMapInformer := kubeInformerFactory.ApiAuthConfigMap()
+
+	k8sCAManager := kvtls.NewKubernetesClientCAManager(authConfigMapInformer.GetStore())
+
+	handlerCertManager := bootstrap.NewFileCertificateManager(virt_api.DefaultHandlerCertFilePath, virt_api.DefaultHandlerKeyFilePath)
+
+	handlerTLSConfiguration := kvtls.SetupTLSForVirtHandlerClients(k8sCAManager, handlerCertManager, false)
```

Comment:
you may have to wait for the informer to sync here when establishing a watch. Otherwise it may not yet have the configmap when the ca manager is asked for the client.

Reply:
Good catch!
Done :+1: